pkg/network/mgmt/flannel.go

```diff
@@ -0,0 +1,53 @@
+package mgmt
+
+import (
+	"fmt"
+
+	"github.com/vishvananda/netlink"
+
+	"github.com/harvester/harvester-network-controller/pkg/network/iface"
+)
+
+const flannelVtepName = "flannel.1"
```

Comment:
can we change this to the management network and make the veth name configurable